python/nav/web/seeddb/page/netbox/edit.py

```diff
@@ -290,6 +290,13 @@ def netbox_do_save(form):
         else:
             func.value = function
         func.save()
+    elif function == '':
+        try:
+            func = NetboxInfo.objects.get(netbox=netbox, variable='function')
+        except NetboxInfo.DoesNotExist:
+            pass
+        else:
+            func.delete()
```

Comment:
Nice. An alternate shorter version could be 
```suggestion
    elif function == '':
        NetboxInfo.objects.filter(netbox=netbox, variable='function').delete()
```